docs/spicedb/feature-overview.md

```diff
@@ -0,0 +1,135 @@
+[API Overview]: /reference/api.md
+
+### Datastore
+
+[Datastores][datastore] are the means of configuring the behavior and performance of persisting relationship and schema data.
+
+To understand the trade-offs between various implementations of Datastores, see the [Selecting a Datastore][select-datastore] guide.
+
+[datastore]: /reference/glossary.md#datastore
+[select-datastore]: spicedb/selecting-a-datastore.md
+
+#### Read-only
+
+All datastores can be configured to operate in a read-only mode.
+When API requests that attempt to perform writes are made while in read-only, the API will return an [Unavailable status][grpc-status] with the reason `SERVICE_READ_ONLY`.
+
+[grpc-status]: https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
+
+#### Hedging
+
+All datastores support having can have their requests hedged.
+If a request to the datastore has taken too long, another request is made to hedge against the first ever completing.
+A configurable quantile value, defaulting to p95, is used to determine whether or not a request is slow.
+When there are not enough samples to accurately use a quantile, a configured absolute duration is used.
+
+### Dispatching
+
+[Dispatching][dispatch] is the primary means of distributing caching across a SpiceDB deployment.
+By default, dispatching only occurs on a single instance.
+When configured with an server to listen for dispatch requests and a source of peer discovery, SpiceDB will use consistent hashing to map specific requests to specific instances in the deployment with the ultimate goal of improving cache hit rates.
+
+[dispatch]: /reference/glossary.md#dispatchers
+
+### Dashboard
+
+The dashboard serves an internal website for humans operating SpiceDB.
+It currently includes simple instructions for how to configure [zed][zed] for the running instance.
+
+[zed]: https://github.com/authzed/zed
+
+## Integration Tests Entrypoint
+
+The integration testing entrypoint runs a modified versions of API service intended for applications to use as a fixture for running tests.
+Each [Bearer Token][bearer-token] provided by the client is allocated its own isolated, ephemeral datastore.
+By generating a unique authentication token for each test, tests can be executed in parallel safely against a single instance of SpiceDB.
+
+[bearer-token]: /reference/api.md#authentication
+
+### GitHub Action
+
+One of the best ways to consume the integration tests entrypoint is via the [SpiceDB GitHub Action][gha].
+
+[gha]: https://github.com/authzed/action-spicedb
+
+## Devtools Entrypoint
+
+The devtools entrypoint is currently used to power the [Playground][playground].
+It has the capability to validate schemas with amounts of relationships and assertions that fit in memory.
+It can also optionally persist and serve bundled schemas with their relationships and assertions such that they can be shared or imported via [zed][zed] or using flags designed to bootstrap SpiceDB.
```

Comment:
```suggestion
It can also optionally persist and serve schemas bundled with their relationships and assertions. These bundles can be shared or imported into SpiceDB via [zed][zed] or using flags for bootstrapping SpiceDB.
```